docs/team/apply.de.md

```diff
@@ -1,6 +1,6 @@
-Ob und wie du dich im Bereich Administration, Development, Moderation, Support, Building oder Content bewerben kannst, erfährst du hier.
+Ob und wie du dich im Bereich Administration, Development, System Administration, Moderation, Support, Building oder Content bewerben kannst, erfährst du hier.
```

Comment:
Am besten, nach der Reihenfolge auf der Tablist.
In dem Fall würde System Administration direkt nach Administration kommen.

Reply:
War jetzt bewusst so, aber wenns in der Tab so ist passe ich das an